persistence-modules/core-java-persistence-2/src/main/java/com/baeldung/microstream/Author.java

```diff
@@ -0,0 +1,38 @@
+package com.baeldung.microstream;
+
+import java.util.Objects;
+
+public class Author {
+
+    private final String name;
+    private final String surname;
+
+    public Author(String name, String surname) {
+        this.name = name;
+        this.surname = surname;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getSurname() {
+        return surname;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o)
+            return true;
+        if (o == null || getClass() != o.getClass())
+            return false;
```

Comment:
**MINOR** - I know this is generated code, but can we add the missing `{}` to the bodies of the `if` statement?